lib/cable_ready/broadcaster.rb

```diff
@@ -5,12 +5,122 @@ module Broadcaster
     include Identifiable
     extend ::ActiveSupport::Concern
 
+    included do |base|
+      if base < ActiveRecord::Base
+        include ExtendHasMany
+        after_commit :broadcast_collections
+        after_create_commit { @cable_ready_current_callback = :create }
+        after_update_commit { @cable_ready_current_callback = :update }
+        after_destroy_commit { @cable_ready_current_callback = :destroy }
+
+        def self.enable_broadcasts(*options)
+          options = options.extract_options!
+          options = {
+            on: [:create, :update, :destroy],
+            if: -> { true }
+          }.merge(options)
+          after_commit :broadcast_self, options
+        end
+      end
+    end
+
     def cable_ready
       CableReady::Channels.instance
     end
 
     def cable_car
       CableReady::CableCar.instance
     end
+
+    private
+
+    def broadcast_self
+      ActionCable.server.broadcast(to_global_id, {})
```

Comment:
Totally naive question, but can we narrow this a bit down to only use the `CableReady:Stream` identifier?

Reply:
Not a bad question, but I suspect you're confusing yourself because `CableReady:Stream` is a *channel* not an identifier.

You'll be happy to know what we **are** using the `CableReady:Stream` channel for both. 

Let me know if this clears things up (or I misunderstood!)

Reply:
no thanks, I'm just still unfamiliar with vanilla `ActionCable` workings :D

Reply:
Hey, I still frequently have to speak about AC at toddler speed in my head, just to make sure I don't lose the thread.

But yeah, I'm a little proud of the degree to which I was able to reuse the StreamFrom functionality to implement this new broadcast stuff.